include/simdjson/portability.h

```diff
@@ -91,16 +79,93 @@ use a 64-bit target such as x64 or 64-bit ARM.")
 //
 // The simdjson users should still have the fallback kernel. It is 
 // slower, but it should run everywhere.
+
+//
+// Enable valid runtime implementations, and select SIMDJSON_BUILTIN_IMPLEMENTATION
+//
+
+//
+// ARM 64-bit (NEON / Other)
+//
+#if SIMDJSON_IS_ARM64
+// Default ARM64 to on: it could be selected at runtime.
+#ifndef SIMDJSON_IMPLEMENTATION_ARM64
+#define SIMDJSON_IMPLEMENTATION_ARM64 1
+#endif
+#if __ARM_NEON
```

Comment:
64-bit ARM always includes NEON so `#if __ARM_NEON` could be omitted.

I think that simdjson should run on all 64-bit ARM processors. There is no need to branch. The only optional features are tiny crypto thingies that we do not use. (We used to but removed them.) That's why here is no runtime dispatch under ARM.